setup.py

```diff
@@ -123,7 +53,6 @@ def _unavailable(self, e):
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
-        "Programming Language :: Python :: Implementation :: CPython",
         "Programming Language :: Python :: Implementation :: PyPy",
```

Comment:
This makes me think this package is PyPy-only and doesn't support CPython anymore.

My gut feeling is that we should either list both CPython and PyPy or neither, and that it's better to list both when we have explicit CI tests on both.